src/components/App/App.jsx

```diff
@@ -22,9 +20,8 @@ class App extends Component {
       <div className="App">
         <div className="menu-header">
           <h2>Welcome to Jordan Sassy Menu </h2>
-          <Menu />
-          <Items name={this.props.items.name} />
```

Comment:
You were probably troubleshooting with these. I took em out cause it isn't needed until later.